engine/gamesys/src/gamesys/scripts/script_tilemap.cpp

```diff
@@ -460,6 +460,74 @@ namespace dmGameSystem
         return TileMap_Get(L, true);
     }
 
+    /*# get all the tiles from a layer in a tilemap
+     * Retrieves all the tiles for the specified layer in the tilemap.
+     * It returns a table of tables where the keys are the
+     * tile positions (see [ref:tilemap.get_bounds()]).
+     *
+     * @name tilemap.get_tiles
+     * @param url [type:string|hash|url] the tilemap
+     * @param layer [type:string|hash] the name of the layer for the tiles
+     * @return tiles [type:table] a table of tables representing the layer
```

Comment:
We should write that the returned table is a table of columns and not a table of rows.

tiles[y][x] vs tiles[x][y]

Personally I prefer tiles[x][y]